.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Description
+A few sentences describing the overall goals of the pull request's commits.
+
+# Checks
+Check all check marks that apply to this pull request. It is not
+necessary check all of them.
+
+- [ ] Minor edit
+- [ ] Doc Update section is added, if not minor edit
+- [ ] Checked if backport applies to maintenance/SLE12
```

Comment:
I am not 100% sure how to interpret these boxes: mebbe use "Backport to ... needed" (if that is a the correct interpretation)?